mobile-browser-based-version/server/src/logic/federated/async_weights_informant.ts

```diff
@@ -0,0 +1,92 @@
+import { AsyncWeightsBuffer } from './async_weights_buffer'
+
+/**
+ * @remarks
+ * taskID: corresponds to the task that weights correspond to.
+ */
+export class AsyncWeightsInformant {
+    taskID: string;
```

Comment:
Since a weight informant is observing an `asyncWeightsBuffer`, he can retrieve the taskID from there? (maybe redundant to also store it here?)

Reply:
Sure thing !

Reply:
Oh Btw I noticed that when creating the `asyncWeightsBuffer` we were passing the whole task object and not the taskId, I'll double check that